README.md

```diff
@@ -3,14 +3,63 @@
 
 Add an email address for use in your [IRMA app](https://github.com/privacybydesign/irma_mobile).
 
-
 ## Setting up the server
 
-1. Generate JWT keys for the issuer
+### Prerequisite
+
+Go, Gradle v4, JDK, Yarn, and an installation of IRMA server:
+
+```bash
+git clone git@github.com:privacybydesign/irmago.git
+cd irmago/
+go install ./irma
+```
+
+### Install
+
+1. Clone repository
+```bash
+git clone git@github.com:privacybydesign/irma_email_issuer.git
+cd irma_email_issuer/
+```
+2. Generate JWT keys for the issuer
 ```bash
 ./utils/keygen.sh ./src/main/resources/sk ./src/main/resources/pk
 ```
-2. Copy the file `src/main/resources/config.sample.json` to
- `build/resources/main/config.json` and modify it.
-3. Run `gradle appRun` in the root directory of this project.
-4. Navigate to `http://localhost:8080/irma_email_issuer/api/hello`
+3. Build the project
+```bash
+gradle build
+```
```

Comment:
I think the `gradle appRun` is enough to run the project, so for that purpose you don't need to explicitly call build if I'm right.

Reply:
yes, I added this step to create the `build` directory tree which we need for the next step.